packages/features/auth/lib/next-auth-options.ts

```diff
@@ -400,6 +400,7 @@ export const AUTH_OPTIONS: AuthOptions = {
           role: user.role,
           impersonatedByUID: user?.impersonatedByUID,
           belongsToActiveTeam: user?.belongsToActiveTeam,
+          organizationId: user?.organizationId,
```

Comment:
Making sure we have the orgId in the session to then rely on it for branding provider